### ISSUE TITLE
[WordPress with NGINX and SSL] The website is Not secure

### ISSUE THREAD

**Issue by Jeep-ECMS** [bitnami/vms#1076]
### Describe your issue as much as you can

I'm subscribed the "WordPress with NGINX and SSL Certified by Bitnami and Automattic"
But it seems to happen that my WP is "Not secure" with the "Certificate is not valid"
Do I need to set more ?
Or
![Screenshot_5](https://github.com/bitnami/vms/assets/129986339/306d86ae-7300-4aad-93d0-166ce21eaf43)
 where can I set it ?

**Comment by jotamartos** [comment]
Did you follow the documentation to generate the SSL certificate?

https://docs.bitnami.com/general/how-to/generate-install-lets-encrypt-ssl/#alternative-approach

**Comment by Jeep-ECMS** [comment]
Sorry, but where can I access to the command ? 
I just install a WP on AWS EC2 and just that, so I cannot know where can I access to the command and generate the let's Encrypt ?
Could you show me more the gateways please, Thanks in advance

**Comment by Jeep-ECMS** [comment]
Thank you for your information, but let me make sure a little bit more.

1st → I have to create a new "key pair" and "instance" on the Lightsail (even I already have them on EC2) ?
2nd → Run the Bitnami command and generate the SSL there?

**Comment by Jeep-ECMS** [comment]
Thank you for your information, but let me make sure a little bit more.

1st → Transfer my old domain to to my public IP site.
2nd → Have to create a new "key pair" and "instance" on the Lightsail (even I already have them on EC2)
3rd → Run the Bitnami command and choose the domain then generate the SSL there?
![Screenshot_3](https://github.com/bitnami/vms/assets/129986339/c4ba5b87-37da-412f-b368-57f8e5bf2138)
![Screenshot_4](https://github.com/bitnami/vms/assets/129986339/409fead8-ad51-48f5-89d1-ef2c0439e342)

**Comment by jotamartos** [comment]
Yes, that's the approach to follow. The command will generate a new SSL certificate in the new instance and validate that the domain is properly configured.

**Comment by Jeep-ECMS** [comment]
Appreciate much for your support, if I have more question I'll ask you again.

**Comment by Jeep-ECMS** [comment]
Sorry for bothering again, but could I ask you for more questions.

My Instance type is t3a.small 2vCPU, 2GiB memory and my Storage is GP2 10GB.

But I still facing the problem of site downs 1 yesterday and another 1 today.
So could you give me some idea to fix this? 

Thank you in advance.

**Comment by jotamartos** [comment]
Please monitor the number of requests your system receives and the memory usage to know when the issue happens. You can also review the Apache's access_log file to know if there is any bot/attacker requesting your site with malicious intentions. 

https://docs.bitnami.com/general/apps/wordpress/troubleshooting/deny-connections-bots-apache/

**Comment by Jeep-ECMS** [comment]
Thank you very much
